src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/javaparser/InputJavaParserSwitchExpressionTest.java

```diff
@@ -0,0 +1,125 @@
+//non-compiled with javac: Compilable with Java12
```

Comment:
Please add a new build configuration to ensure that this file is valid java code.
Al least, for java 12 compiler.

See https://github.com/checkstyle/checkstyle/blob/master/.travis.yml#L231 https://github.com/checkstyle/checkstyle/blob/master/.ci/travis/travis.sh#L69 for the reference.